packages/connect-explorer/src/store/index.ts

```diff
@@ -1,12 +1,12 @@
 import { applyMiddleware, compose, createStore } from 'redux';
 import { createLogger } from 'redux-logger';
-import thunk from 'redux-thunk';
+import { thunk } from 'redux-thunk';
 
 import { trezorConnectMiddleware } from '../middlewares/trezorConnectMiddleware';
 import { reducers } from '../reducers';
 
 const enhancers: any[] = [];
-const middleware = [thunk, trezorConnectMiddleware];
+const middleware: any[] = [thunk, trezorConnectMiddleware];
```

Comment:
Figured it out to make it somewhat type-safer. Otherwise, wouldn't the `any` type corrupt the types downstream?

```suggestion
const middleware = [thunk, trezorConnectMiddleware] as Middleware<Dispatch, AppState>[];
```

This would be even better
```
const middleware: Middleware<Dispatch, AppState>[] = [thunk, trezorConnectMiddleware];
```

but it says this and I don't know how to solve that anymore..
```
Call signature return types (next: Dispatch) => (action: Action) => void and (next: (action: unknown) => unknown) => (action: unknown) => unknown are incompatible.
Type (action: Action) => void is not assignable to type (action: unknown) => unknown
Types of parameters action and action are incompatible.
Type unknown is not assignable to type Action
```

Reply:
I didn't know either that reason for `any[]` but I don't think it matter here it won't affect type safety in anyway becacause this type is used only once in configureStore